skipruntime-ts/examples/utils.ts

```diff
@@ -25,7 +25,7 @@ export class SkipHttpAccessV1 {
 
   constructor(
     private streaming_port: number = 8080,
-    private control_port: number = 8081,
+    control_port: number = 8081,
```

Comment:
Is this change needed?

Reply:
Yes, otherwise that's recognized as an unused field by our JS tooling.  Without `private`, it's just a local variable argument of the constructor.